kpatch-build/kpatch-build

```diff
@@ -447,64 +447,64 @@ mkdir -p "$TEMPDIR" || die "Couldn't create $TEMPDIR"
 rm -rf "$TEMPDIR"/*
 rm -f "$LOGFILE"
 
-if [[ -n $USERSRCDIR ]]; then
+if [[ -n "$USERSRCDIR" ]]; then
 	SRCDIR="$USERSRCDIR"
 
-	[[ -z $VMLINUX ]] && VMLINUX="$SRCDIR"/vmlinux
+	[[ -z "$VMLINUX" ]] && VMLINUX="$SRCDIR"/vmlinux
 	[[ ! -e "$VMLINUX" ]] && die "can't find vmlinux"
 
 	# Extract the target kernel version from vmlinux in this case.
-	ARCHVERSION=$(strings "$VMLINUX" | grep -e "^Linux version" | awk '{ print($3); }')
+	ARCHVERSION="$(strings "$VMLINUX" | grep -e "^Linux version" | awk '{ print($3); }')"
 fi
 
-[[ $SKIPCLEANUP -eq 0 ]] && trap cleanup EXIT INT TERM HUP
+[[ "$SKIPCLEANUP" -eq 0 ]] && trap cleanup EXIT INT TERM HUP
 
-KVER=${ARCHVERSION%%-*}
-if [[ $ARCHVERSION =~ - ]]; then
-	KREL=${ARCHVERSION##*-}
-	KREL=${KREL%.*}
+KVER="${ARCHVERSION%%-*}"
+if [[ "$ARCHVERSION" =~ - ]]; then
+	KREL="${ARCHVERSION##*-}"
+	KREL="${KREL%.*}"
 fi
 
-[[ -z $TARGETS ]] && TARGETS="vmlinux modules"
+[[ -z "$TARGETS" ]] && TARGETS="vmlinux modules"
 
 source /etc/os-release
-DISTRO=$ID
-if [[ $DISTRO = fedora ]] || [[ $DISTRO = rhel ]] || [[ $DISTRO = ol ]] || [[ $DISTRO = centos ]]; then
-	[[ -z $VMLINUX ]] && VMLINUX=/usr/lib/debug/lib/modules/$ARCHVERSION/vmlinux
+DISTRO="$ID"
+if [[ "$DISTRO" = fedora ]] || [[ "$DISTRO" = rhel ]] || [[ "$DISTRO" = ol ]] || [[ "$DISTRO" = centos ]]; then
+	[[ -z "$VMLINUX" ]] && VMLINUX="/usr/lib/debug/lib/modules/$ARCHVERSION/vmlinux"
 	[[ -e "$VMLINUX" ]] || die "kernel-debuginfo-$ARCHVERSION not installed"
```

Comment:
`DISTRO` doesn't need quoting (here and below)

Reply:
Dito.